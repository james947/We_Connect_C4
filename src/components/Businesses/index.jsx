import React from 'react';
import FetchBusinesses from '../../containers/FetchBusinesses';
import Footer from '../Footer/Footer';
import NavBar from '../NavBar/NavBar';
import SearchBusiness from '../../containers/SearchBusiness';

const Businesses = () => (
  <div>
    <NavBar />
    <div className="container">
        <div className="col-md-10">
        <SearchBusiness/>
          <FetchBusinesses />
        </div>
    </div>
    <Footer />
  </div>
);

export default Businesses;
