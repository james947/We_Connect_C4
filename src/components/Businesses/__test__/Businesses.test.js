import { shallow } from 'enzyme'
import React from 'react';
import  Businesses  from '../';

describe("<Businesses/>", () => {
    it('renders correctly', () => {
        const wrapper = shallow(<Businesses/>);
        expect(wrapper).toMatchSnapshot();
    })
})