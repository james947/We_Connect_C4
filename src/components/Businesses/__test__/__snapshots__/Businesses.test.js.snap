// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Businesses/> renders correctly 1`] = `
<div>
  <Connect(NavBar) />
  <div
    className="container"
  >
    <div
      className="col-md-10"
    >
      <Connect(SearchPage) />
      <Connect(FetchBusinesses) />
    </div>
  </div>
  <Footer />
</div>
`;
